src/components/Popups/PopupBase.scss

```diff
@@ -9,6 +9,13 @@
     .ba-Popup {
         z-index: 1;
 
+        .btn.btn-primary {
```

Comment:
Should thse styles live in a PopupReply-specific stylesheet? The PopupBase component doesn't contain any buttons.